store-server/store/apps/shop/models.py

```diff
@@ -174,6 +183,11 @@ class SaleBanner(models.Model):
     def __str__(self):
         return str(self.subject)
 
+    def save(self, *args, **kwargs):
+        if SaleBanner.objects.count() >= settings.MAX_BANNER_COUNT:
```

Comment:
I see a lot of different banners here, create `abstract` model for it or if they are similar create one model with field `type`